pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -77,8 +77,76 @@ type MonitoringStackSpec struct {
 // MonitoringStackStatus defines the observed state of MonitoringStack.
 // It should always be reconstructable from the state of the cluster and/or outside world.
 type MonitoringStackStatus struct {
-	// TODO(sthaha): INSERT ADDITIONAL STATUS FIELDS -- observed state of prometheus
-	// ??
+	// Conditions provide status information about the MonitoringStack
+	// +listType=map
+	// +listMapKey=type
```

Comment:
I assume that @tremes took it from the Prometheus Operator;)
I don't think that it matters much since as you said, 1) the controller always updates the status (no patch) and 2) nobody else should be modifying the resource status.

Reply:
Fair points! I would still argue for the "correct" markers if only to inform future developers of the intention.

Reply:
OK. Thanks